html-tables/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Tables</title>
+  </head>
+  <body>
+    <h1>Student Grade Table</h1>
+    <table>
+      <thead>
+        <tr>
+          <th scope="col">Student Name</th>
+          <th scope="col">Course</th>
+          <th scope="col">Grade</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td scope="row">Grace Hopper</td>
+          <td scope="row">Computer Science</td>
+          <td scope="row">100</td>
+        </tr>
+        <tr>
+          <td scope="row">Alan Turing</td>
+          <td scope="row">Mathematics</td>
+          <td scope="row">100</td>
+        </tr>
+        <tr>
+          <td scope="row">Douglas Crockford</td>
+          <td scope="row">JSON</td>
+          <td scope="row">98</td>
+        </tr>
+        <tr>
+          <td scope="row">Tim Berners-Lee</td>
+          <td scope="row">HTML</td>
+          <td scope="row">9001</td>
+        </tr>
+        <tr>
+          <td scope="row">Linus Torvalds</td>
+          <td scope="row">Git</td>
+          <td scope="row">90</td>
```

Comment:
The `scope` attribute is unnecessary here, it is only pertinent to `th` elements